src/model/ModelObject.hpp

```diff
@@ -287,21 +285,31 @@ class MODEL_API ModelObject : public openstudio::WorkspaceObject {
   REGISTER_LOGGER("openstudio.model.ModelObject");
 };
 
-class MODEL_API EMSActuatorNames
-{
-public:
-
+class MODEL_API EMSActuatorNames {
+ public:
   EMSActuatorNames(const std::string & componentTypeName, const std::string & controlTypeName);
 
   std::string controlTypeName() const;
   std::string componentTypeName() const;
 
-private:
-
+ private:
   std::string m_componentTypeName;
   std::string m_controlTypeName;
 };
 
+/** Simple class for ScheduleTypeRegistry key. \relates ModelObject */
+class MODEL_API ScheduleTypeKey {
```

Comment:
OK I see you changed the type of ScheduleTypeKey from std::pair. But this is a breaking API change. I wouldn't complain except I also saw that the std::pair was even deliberately swigged. So we shouldn't take this change lightly. Is it the only way? If so then at least we should say something in release notes.

Reply:
Probably not _mandatory_, but much cleaner, and it does fix a C# bindings issue with an improperly wrapped type. There are multiple occasions where `std::pair<std::string, std::string>` was defined to something that was clashing too if I recall correctly.

Like `src/utilities/core/Containers.hpp:58:typedef std::pair<std::string,std::string> StringPair;>`

Adding it to the release notes would make sense.

I doubt anyone uses `ScheduleTypeKey` much anyways, this stuff is meant to be used by the ScheduleRegistry internally, not messed with.

Reply:
I agree on all of your points. Can you just add a comment in the release notes and then I'll just go ahead and merge? I think you will have to start a new file https://github.com/NREL/OpenStudio/tree/develop/developer/doc/ReleaseNotes for version 3.1.

Reply:
Done

Reply:
https://github.com/NREL/OpenStudio/pull/3960/files#diff-7f9f2001055db45a596d70f0e33579ce